### DISCUSSION TITLE
undesired export result in bib file

### DISCUSSION THREAD

**Discussion by CamuseCao** [retorquere/zotero-better-bibtex#2958] [Q&A] [answered]
I'm using zotero 7.0.0-1 with BBT 6.7.222. I had a item with title
 `Bandwidth control in a perovskite-type ${3d}^{1}$-correlated metal ${\mathrm{Ca}}_{1\ensuremath{-}x}{\mathrm{Sr}}_{x}{\mathrm{VO}}_{3}.$ I. Evolution of the electronic properties and effective mass', 
which is auto generated by drop a pdf into zotero. It's exported to biblatex and get the result in the fig.

![image](https://github.com/user-attachments/assets/5b74241b-6a06-4a28-b0fa-bcf9e9d4df5b)

I don't want those  `$ \mathrm` and so on to be export to `\$ \textbackslash mathrm`. What should I do?
It seems like only happen in the title.
I have checked settings  several times, however, find nothing relating to it.

**Comment by retorquere** [comment]
Please update both Zotero and BBT.

It shouldn't only happen in the title, but if you don't tell me where it does not happen where you expected it to happen, I can't tell you what's going on.

WRT to the exported `\$` etc, in general terms, the data you see in Zotero is just plain text, not LaTeX, so a `$` character in Zotero just means a dollar sign, not "start of math mode", and so on export, BBT exports it in a way that means "dollar sign", which is `\$`. If you want it to be interpreted as such, you can try it with a [postscript](https://retorque.re/zotero-better-bibtex/exporting/scripting/index.html#detect-and-protect-latex-math-formulas).

> which is auto generated by drop a pdf into zotero. It's exported to biblatex and get the result in the fig.

BBT is not involved in this. If you want the behavior to be different you will have to ask on the Zotero forums.

For further help on my end, you will need to right-click the item under discussion, select "Better BibTeX" -> "Debug log", and post the debug log ID here. It's not efficient for me to guess what is in your library that causes the behavior you want changed.

**Reply by CamuseCao** [reply]
Now I have already updated Zotero to  7.0.3_beta.1_2065a60b7-1 and BBT 6.7.225. Still get the same result.

When I want to send debug log, I got the following error message `Error: Request timed out (DZ4NDCFG-apse, items: false)`.
I'm not so sure whether the log is sent or not.

**Reply by CamuseCao** [reply]
`953NUJ96-apse/6.7.225-7` is the debug log ID.

**Reply by retorquere** [reply, marked as answer]
Whether that works for you depends on whether you exported as BibTeX (not targeted by your current postscript) or BibLaTeX (which your postscript explicitly picks out). If you want it to work for both, change `BetterBibLaTeX` into `BetterTeX`.

**Reply by CamuseCao** [reply]
After change the `Translator.BetterBibLaTeX` to `Translator.BetterTeX` in the postscript, it works for me. Thanks for your help.
